release-notes/5.0/preview/api-diff/preview2/.Net/5.0-preview2_System.Text.RegularExpressions.md

```diff
@@ -0,0 +1,15 @@
+# System.Text.RegularExpressions
+
+``` diff
+ namespace System.Text.RegularExpressions {
+     public class Regex : ISerializable {
+-        protected bool UseOptionR();
++        protected internal bool UseOptionR();
```

Comment:
This isn't actually a change. There's no difference between `protected` and `protected internal` from an external consumption standpoint; the `internal` piece of it is an implementation detail.

Reply:
Api reviewer is not smart enough to  detect this. Do u have any preference towards removing this and https://github.com/dotnet/core/pull/4506#discussion_r403195941 ?

Reply:
I'd prefer we not include such changes in the diff, as there's actually no API change here.